Makefile

```diff
@@ -56,6 +56,12 @@ $(TEST_DIR)/%.o: $(TEST_DIR)/%.c $(BIN) $(OBJ_DIR)/$(BIN)
 	$(Q)$(ARM_EXEC) ./$(OBJ_DIR)/$(BIN) $< 2 $(REDIR)
 	$(Q)$(call pass,$<)
 
-clean:
+help: ## Prints help for targets with comments
+	@cat $(MAKEFILE_LIST) | grep -E '^[a-zA-Z_-]+:.*?## .*$$' | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+dump_asm:$(BIN) ## Dump assembly from source file,usage:make dump_asm FILE=tests/main.cc
```

Comment:
Rename this target to `dump-ir`, which generates IR instead of Arm machine code.